core/src/main/java/org/springframework/security/core/Authentication.java

```diff
@@ -134,4 +134,8 @@ public interface Authentication extends Principal, Serializable {
 	 */
 	void setAuthenticated(boolean isAuthenticated) throws IllegalArgumentException;
 
+	default String getAuthenticationType() {
```

Comment:
While this is convenient, I'd prefer not to add extra noise to the interface. If this was really something that was necessary, I'd expect that Object had a convenience method for this (which doesn't make sense to me).

Reply:
It's less about convenience (for me) and more about allowing authentication implementations to configure what gets published to Micrometer as the authentication type.

That said, I think this was added fairly early on before I understood Micrometer's convention API. I'll remove this and we can make customizing the observation convention a possibility in a future release.